main.go

```diff
@@ -98,18 +98,21 @@ func main() {
 	// define environment
 	var env gateways.Environment
 	var confFileName string
+	var configURL string
 	if ENV == "prod" {
 		env = gateways.Production
 		zerolog.SetGlobalLevel(zerolog.InfoLevel)
 		confFileName = "config.yaml"
+		configURL = "https://device-config-prod.dimo-builder.workers.dev/"
```

Comment:
@JamesReate , these are the current cloudflare routes, we might change them but no other code change required.